cli/src/commands/git/init.rs

```diff
@@ -252,3 +258,32 @@ pub fn maybe_set_repository_level_trunk_alias(
 
     Ok(())
 }
+
+fn track_local_bookmarks(ui: &mut Ui, repo: &mut MutableRepo) -> Result<(), CommandError> {
+    let pairs: Vec<(String, String)> = repo
+        .view()
+        .bookmarks()
+        .filter(|(_, bookmark_target)| bookmark_target.local_target.is_present())
+        .flat_map(|(name, bookmark_target)| {
+            bookmark_target
+                .remote_refs
+                .into_iter()
+                .filter(|&(remote_name, _)| remote_name != "git")
+                .map(|(remote_name, _)| (name.into(), remote_name.into()))
```

Comment:
nit: Use the more specific (and therefore clearer) `.to_owned()` or `to_string()` here. Feel free to ignore if you like `.into()` better.